tox.ini

```diff
@@ -100,9 +100,9 @@ envlist =
     ; py310 does not work with falcon 1
     ; The numbers at the end of the environment names
     ; below mean these dependencies are being used:
-    ; 0: falcon ==1.4.1
-    ; 1: falcon >=2.0.0,<3.0.0
-    ; 2: falcon >=3.0.0,<4.0.0
```

Comment:
I thought the goal of these comments was to preserve the actual range so when it comes time to upgrade the pinned version, we can make sure it meets this requirement?